client/packages/common/src/intl/utils/DateUtils.ts

```diff
@@ -45,6 +47,15 @@ export const DateUtils = {
   isAfter,
   isBefore,
   isEqual,
+  parseNaiveDateTime: (dateTime?: string): Date | null => {
+    if (!dateTime) return null;
+
+    const maybeDate = dateTime.endsWith('Z')
```

Comment:
the reason that I was always seeing 13h ago.. the datetime that the server is sending is a string without any timezone. This is picked up by the f/e as being local 🙄 

Perhaps should be corrected in the server? Unless there is a reason - I seem to remember timezone being attached at one time?

Reply:
oops, it should have been  DateTime<Utc> like other api result: https://github.com/openmsupply/open-msupply/compare/688-manual-sync-extra...688-example-datetime-utc

Reply:
ah, thanks. am merging it in and removing the `DateUtils` change